integrations/azure/.port/resources/blueprints.json

```diff
@@ -0,0 +1,447 @@
+        "deleted": {
+          "title": "Deleted",
+          "type": "boolean"
+        },
+        "deletedTime": {
+          "title": "Deleted Time",
+          "type": "string"
+        },
+        "remainingRetentionDays": {
+          "title": "Remaining Retention Days",
+          "type": "number"
+        },
+        "leaseStatus": {
+          "title": "Lease Status",
+          "type": "string"
+        },
+        "leaseState": {
+          "title": "Lease State",
+          "type": "string"
+        },
+        "defaultEncryptionScope": {
+          "title": "Default Encryption Scope",
+          "type": "string"
+        },
+        "version": {
+          "title": "Version",
+          "type": "string"
+        }
+      },
+      "required": []
+    },
+    "mirrorProperties": {},
+    "calculationProperties": {},
+    "relations": {
+      "storageAccount": {
+        "target": "storageAccount",
+        "title": "Storage Account",
+        "required": true,
+        "many": false
+      }
+    }
+  },
+  {
+    "identifier": "virtualMachine",
+    "description": "This blueprint represents an Azure Virtual Machine in our software catalog",
+    "title": "Virtual Machine",
+    "icon": "Azure",
+    "schema": {
+      "properties": {
+        "location": {
+          "title": "Location",
+          "type": "string"
+        },
+        "provisioningState": {
+          "title": "Provisioning State",
+          "type": "string"
+        },
+        "vmSize": {
+          "title": "VM Size",
+          "type": "string"
+        },
+        "osDiskName": {
+          "title": "OS Disk Name",
+          "type": "string"
+        },
+        "osDiskType": {
+          "title": "OS Disk Type",
+          "type": "string"
+        },
+        "osDiskCaching": {
+          "title": "OS Disk Caching",
+          "type": "string"
+        },
+        "osDiskSizeGB": {
+          "title": "OS Disk Size GB",
+          "type": "number"
+        },
+        "osDiskCreateOption": {
+          "title": "OS Disk Create Option",
+          "type": "string"
+        },
+        "networkInterfaceIds": {
+          "title": "Network Interface IDs",
+          "type": "array"
+        },
+        "licenseType": {
+          "title": "License Type",
+          "type": "string"
+        },
+        "vmOsProfile": {
+          "title": "VM OS Profile",
+          "type": "object"
+        },
+        "vmHardwareProfile": {
+          "title": "VM Hardware Profile",
+          "type": "object"
+        },
+        "vmStorageProfile": {
+          "title": "VM Storage Profile",
+          "type": "object"
+        },
+        "tags": {
+          "title": "Tags",
+          "type": "object"
+        }
+      },
+      "required": []
+    },
+    "mirrorProperties": {},
+    "calculationProperties": {},
+    "relations": {
+      "resourceGroup": {
+        "target": "resourceGroup",
+        "title": "Resource Group",
+        "required": true,
```

Comment:
I would avoid making the relation required (across all BPs) it could create some additional complexity for the user when deleting entities